src/renderer/components/Settings/ManageKeys.tsx

```diff
@@ -8,15 +8,21 @@ import useTranslationFunction from '../../hooks/useTranslationFunction'
 import useConfirmationDialog from '../../hooks/useConfirmationDialog'
 
 import type { OpenDialogOptions } from 'electron'
+import { cachedLastUsedPath } from '../../utils/cachedLastUsedPath'
+import { dirname } from 'path'
 
 export default function ManageKeys() {
   const tx = useTranslationFunction()
   const openConfirmationDialog = useConfirmationDialog()
 
   const onKeysImport = useCallback(async () => {
+    const { defaultPath, setLastPath } = cachedLastUsedPath(
+      'last_directory:keys_import',
```

Comment:
Could be nice to have the keys part centrally defined somewhere as constants, then there's a central place where you can see what keys exist in the store, maybe even directly as a type?

How about moving it into the same file, right next to where you define `cachedLastUsedPath`?

Reply:
thought about it, but thought it wouldn't make a big difference, because it is only in memory anyway, but I agree it removes potential of messing up, I can make a type or constant enum for it.